README.md

```diff
@@ -230,25 +233,34 @@ Furthermore, edge weights can be provided by adding a third column to the file `
 
 ## Compilation
 
-The source code can be compiled as follows:
+You must have JDK 1.8+ installed to compile.
+Having Gradle installed is optional as the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) is also included in this repository.
+On Windows, the source code can be compiled as follows:
+
+```
+gradlew build
+```
+
+On Linux and MacOS, use the following command:
 
 ```
-javac -d build src/cwts/networkanalysis/*.java src/cwts/networkanalysis/run/*.java src/cwts/util/*.java
+./gradlew build
```

Comment:
This file is not marked as executable. Could you update the file permissions to ensure it is executable? I believe that is also tracked in GitHub. If not, we should perhaps clarify that people should give access (i.e. `chmod +x gradlew`).

Reply:
I'm working on Windows. Could you perhaps try to change the file permissions?

Reply:
Done